### ISSUE TITLE
Oversize Drawer Slide Option

### ISSUE THREAD

**Issue by TedClassy1209** [CCDevNelson/Snap-Updates#223]
I'm thinking we need an option for each drawer for a manual override to go with (1) the slide size just above the box size rather than (2) the one equal to or below it (how it currently works as I am understanding). I'm thinking we should have an option that goes on the right side of the drawer prompts that says "Oversize Slides" and should be available for each drawer box, which will do what I just wrote above. This will affect the drilling for that drawer slide as well, so I'm not sure how complicated that will get? I don't think it should be bad though? Let me know what you think.

![image](https://user-images.githubusercontent.com/61025894/99425021-947cbf00-28bf-11eb-8a4b-dbd6aa0f09ac.png)

**Comment by ClassyTeddy** [comment]
So when you say it will affect the drilling, how will the oversized slide affect the drilling. At the moment, each size of slide comes with a front hole distance and a back hole distance. Will those change if it is an oversized slide?

**Comment by TedClassy1209** [comment]
> So when you say it will affect the drilling, how will the oversized slide affect the drilling. At the moment, each size of slide comes with a front hole distance and a back hole distance. Will those change if it is an oversized slide?

No, just ensuring that the correct drilling info is passed on for the particular slide that is selected. I don't know how the code currently works for the drilling info--if it is based on the box size or the chosen slide. If it is based on the chosen slide, then it makes it easier I guess.

**Comment by ClassyTeddy** [comment]
So each slide size for each different style of slide has an associated front hole location and a back hole location.

**Comment by TedClassy1209** [comment]
Right, so if I have a 17" deep drawer box and I select that I want the oversized slide (which will give me an 18" slide instead of the 16"), I'm just pointing out that we need to make sure the drilling info exported will be for that 18" slide rather than what would have been the default 16" slide.

**Comment by ClassyTeddy** [comment]
Got it. It should do that once the option to pick the larger slide is added. 
I just wanted to make sure that the slides didn't need different drilling information if they were oversized.

**Comment by TylerStandage** [comment]
@TedCC1209 did we get this in already or is there more updating to do?

**Comment by TedCC1209** [comment]
I don't think this was addressed but I think we need some info first:

1. Is there extra drilling info for drawer slides actually being added to the partitions? If not, it's okay, I just wanted to know
2. I think the default should be that the slide exported for drawers that have extra rear gap is oversized to just be what the default drawer slide would be with no extra rear gap
